browser/resources/settings/sources.gni

```diff
@@ -127,6 +129,7 @@ brave_settings_ts_definitions = [
   "//brave/components/definitions/chrome_brave_theme.d.ts",
   "//brave/components/definitions/chrome_ipfs.d.ts",
   "//brave/components/definitions/chrome_brave_wallet.d.ts",
+  "//brave/components/definitions/chrome_leo.d.ts",
```

Comment:
I think if you use mojo we can get rid of this :smile:

Reply:
Removed it https://github.com/brave/brave-core/pull/19212/commits/414fb6cd24fb8e0b6dfbf0eefabbcb9392065272#diff-4425e803da30d8921e4bbfb14b600b472d493fc297d33896807b8c56a6a6103aL1-L12
and all extension's related code